index.html

```diff
@@ -919,6 +919,93 @@ <h4>
   reader.scan({ recordType: "example.com:sp"});
       </pre>
     </aside>
+    <aside title="Push a smart poster message" class="example">
+      <p>
+        Smart poster defined in NFC Forum Smart Poster Record Type Definition
+        specification. It is an NDEF record that contains an <a>NDEF message</a>
+        as payload, which in turn may contain several records: a URI record that
+        refers to a content, and additional optional records for that content's
+        title, icon, size, type and possible action (e.g. send SMS, make call,
+        launch browser, or save for later, or open for editing, etc).
+      </p>
+      <pre class="highlight">
+        const writer = new NFCWriter();
+        let sp = new NDEFMessage({
+          records: [
+            {
+              recordType: "url",
+              data: "https://my.org/content/19911"
+            },
+            {
+              recordType: "t", // type, a local type to Sp
+              data: "image/gif"
+            },
+            {
+              recordType: "text",
+              data: "Funny dance"
+            },
+            {
+              recordType: "s",  // size, a local type to Sp
+              data: 4096  // byte size of the content at the URL above
+            },
+            {
+              recordType: "act",  // action, a local type to Sp
+              data: 0  // do the action, in this case open in the browser
+            },
+          ];
+        });
+        writer.push({ records: [
+          {
+            recordType: "smart-poster",
+            data: sp.records
+          }
+        ]}).catch(_ => {
+          console.log("Push failed");
+        });
+      </pre>
+    </aside>
+    <aside title="Push an external record with an NDEF message as payload"
+           class="example">
+      <p>
+        External type records can be used to create application defined records
+        that may even contain an <a>NDEF message</a> as payload.
+      </p>
+      <p>
+        This example show how to do the previous Smart Poster use case with
```

Comment:
I find it confusing to refer to Smart Poster here as you are not storing anythign like a smart poster

Reply:
OK, I can skip that remark. What I meant is that you can share the same content in 2 ways: with the standardized Sp, or in an app-specific way using external records and embedded NDEF message.